balancer/balanceraggregator/balanceraggregator.go

```diff
@@ -49,11 +49,12 @@ type ChildState struct {
 
 // NewBalancer returns a new balancerAggregator.
 func NewBalancer(cc balancer.ClientConn, opts balancer.BuildOptions) balancer.Balancer {
-	return &balancerAggregator{
+	ba := &balancerAggregator{
 		cc:       cc,
 		bOpts:    opts,
-		children: resolver.NewEndpointMap(),
 	}
+	ba.children.Store(resolver.NewEndpointMap())
```

Comment:
https://github.com/golang/go/issues/63289 :(

Reply:
Ah :(. Yeah it felt weird having to declare an local var to store a pointer instead of inlining it at struct creation time.